java/src/main/java/ai/rapids/cudf/HostColumnVectorCore.java

```diff
@@ -121,7 +121,7 @@ public long getNullCount() {
   /**
    * Returns the list of child host column vectors for a given host side column
    */
-  List<HostColumnVectorCore> getNestedChildren() {
+  public List<HostColumnVectorCore> getNestedChildren() {
```

Comment:
We already have getChildColumnViewAccess and the list we are returning could be modified by a caller.  If we do want to expose this we need to make a copy of the List, or make it immutable, and then I would not want us to call this internally because it would slow things down.

Reply:
I am sorry I forgot I added that!!!! 
I will close this actually here in a bit.